vinca/main.py

```diff
@@ -1197,8 +1197,6 @@ def main():
     base_dir = os.path.abspath(arguments.dir)
     vinca_yaml = os.path.join(base_dir, "vinca.yaml")
     vinca_conf = read_vinca_yaml(vinca_yaml)
-    snapshot = vinca_conf.get("snapshot", None)
-    additional_packages_snapshot = vinca_conf.get("_additional_packages_snapshot", None)
```

Comment:
Are this changes intentional?

Reply:
Mhh, interesting. That seems a regression from https://github.com/RoboStack/vinca/commit/42709af0b59839a889e3d5662ef4c376aef8560b, but I am not sure if the right thing to do is to add back the use of snapshot or additional_packages_snapshot  that were removed there, but I think that without more test coverage is difficult to say. I guess if necessary we can add it back, as anyhow the regression is not related to this PR.